doc/source/conf.py

```diff
@@ -406,12 +406,6 @@
         "statsmodels": ("http://www.statsmodels.org/devel/", None),
     }
 
-# extlinks alias
-extlinks = {
-    "issue": ("https://github.com/pandas-dev/pandas/issues/%s", "GH"),
```

Comment:
this is used! its how the links in the docs show up (wiki may *not* be used though)

Reply:
Oh, I tried to build the docs locally and I succeeded doing so, can you point me to where it used?
I just want to remove the use of ```%``` strings formmating.

Reply:
when u do :issue: it replaces with this string

Reply:
Oh wow, this is **__LITERALLY__** everywhere.
I really don't want to risk it.

@jreback do you think I should close this PR?

Reply:
yes, though in general removing unused code is encouraged.  i like to use https://codecov.io/gh/pandas-dev/pandas/tree/master/pandas to find unused code (unfortunately it is out of date, hopefully thatll be fixed by #30070)

Reply:
Thank you so much for the tip @jbrockmendel